pkg/kubernetes/kubernetes_test.go

```diff
@@ -0,0 +1,187 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package kubernetes
+
+import (
+	"testing"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	"github.com/stretchr/testify/assert"
+
+	versioned "github.com/dapr/dapr/pkg/client/clientset/versioned"
+	fake "k8s.io/client-go/kubernetes/fake"
+)
+
+const (
+	name        = "test"
+	namespace   = "testns"
+	label       = "testlabel"
+	servicename = "testservice"
+)
+
+var fakeClient = fake.NewSimpleClientset()
+var client = NewTestClients()
```

Comment:
Do we define them globally? the previous comments are to not define the global vars.

Reply:
@youngbupark , I mentioned in the reply that I have it changed to NewTestClient() way but still I need to make it global so to all the test can share it to avoid newing in every test case.

Let me know how can we avoid that thing.